frontend/src/features/googleMap/MapDrawer.tsx

```diff
@@ -36,25 +33,18 @@ const MapDrawer = () => {
     );
   }, []);
 
-  const side = "bottom";
-
-  const defaultCenter = {
-    lat: 35.69575,
-    lng: 139.77521,
-  };
-
   return (
-    <Sheet key={side}>
-      <SheetTrigger asChild>
-        <Button>
-          <Menu />
-        </Button>
-      </SheetTrigger>
-      <SheetContent side={side} className="flex flex-col min-w-screen h-3/4 ">
-        <SheetHeader>地図</SheetHeader>
-        <Map currentLocation={currentLocation} defaultCenter={defaultCenter} />
-      </SheetContent>
-    </Sheet>
+    <div
+      className={`transition-all duration-300 ease-in-out overflow-hidden flex flex-col h-full ${
+        props.show ? "w-[33.33vw]" : "w-0"
+      }`}
```

Comment:
clsxで書き換えましょう！このままでも動くけどね

```suggestion
      className={clsx('transition-all duration-300 ease-in-out overflow-hidden flex flex-col h-full', 
        props.show ? "w-[33.33vw]" : "w-0"
      )}
```